src/components/MenuPanel/MenuPanel.js

```diff
@@ -106,7 +106,6 @@ class MenuPanel extends Component<Props, ComponentState> {
           addressNames={addressNames}
           activeWalletId={activeWalletId}
           isActive={isWalletManagerActive}
-          isReadOnly={isReadOnly}
```

Comment:
because this prop could be obtained from wallet within the [component](https://github.com/jibrelnetwork/jwallet-web/pull/601/files/52d2f36f0580b462c1669ba0da53f4c0aeee7185#diff-a47163115b48fb3de5ef8da6c3a4dc36R44)